app/modules/intelligence/agents/chat_agents/qna_chat_agent.py

```
@@ -2,8 +2,14 @@
 import logging
 import time
 from functools import lru_cache
-from typing import AsyncGenerator, Dict, List
+from typing import Any, AsyncGenerator, Dict, List, Optional
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import

(F401)